### DISCUSSION TITLE
Change/configure hotkeys?

### DISCUSSION THREAD

**Discussion by BRTPOB** [audacious-media-player/audacious#1499] [General]
With Winamp, I was able to use Alt+3 as a hotkey to open a song's metadata so I could edit it. I've set that with foobar2000 when using Windows, but Audacious seems to hardcoded to alt+i. Is there any way I can set that hotkey? I don't need/want it to be a global hotkey, because that's just overkill and not the behavior I"m looking for.

Thanks!

**Comment by radioactiveman** [comment]
Unfortunately this is not possible.
